### PR TITLE
Modify user interface to show stars for Ratings

### LINKED ISSUE #152: Modify user interface to show stars for Ratings

### PR DESCRIPTION
**Pull request by Hikoya**
Resolve #152

### CODE REVIEW

**Comment by jamietan2002** [comment]
LGTM. Good job!

File: src/main/resources/view/StallListCard.fxml [hunk 1]
```diff
@@ -7,7 +7,7 @@
 <?import javafx.scene.layout.RowConstraints?>
 <?import javafx.scene.layout.VBox?>
 
-<HBox id="cardPane" fx:id="cardPane" prefHeight="96.0" prefWidth="350.0" xmlns="http://javafx.com/javafx/11" xmlns:fx="http://javafx.com/fxml/1">
+<HBox id="cardPane" fx:id="cardPane" prefHeight="96.0" prefWidth="350.0" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
```
Comment (clarence-chew) [inline src/main/resources/view/StallListCard.fxml:10]: I see this change - I'm not sure why is this changed to JavaFX 8.

**Review by clarence-chew — COMMENTED** [review]
Looks good, but I have some things I would like to ask.

File: src/main/resources/view/StallListCard.fxml (hunk 1 above)
Comment (Hikoya) [reply src/main/resources/view/StallListCard.fxml:10]: This is for standardization purposes, as all other FXML in the existing project uses JavaFX 8.

**Review by MinHeinA — APPROVED** [review]
LGTM! Thank you for your hard work.
